tests/test_populate_catalog.py

```diff
@@ -136,28 +169,384 @@ def should_update_existing_root_catalog(subtests: SubTests) -> None:
             key=CATALOG_KEY,
         ):
 
-            expected_links: JsonList = original_links + [
+            expected_root_links: JsonList = original_links + [
                 {
                     STAC_REL_KEY: STAC_REL_CHILD,
                     STAC_HREF_KEY: f"./{dataset.dataset_prefix}/{CATALOG_KEY}",
                     STAC_TYPE_KEY: "application/json",
                 }
             ]
 
-            body = {RECORDS_KEY: [{"body": dataset.dataset_prefix}]}
+            expected_dataset_links: JsonList = [
+                {
+                    STAC_REL_KEY: STAC_REL_ROOT,
+                    STAC_HREF_KEY: f"../{CATALOG_KEY}",
+                    STAC_TYPE_KEY: "application/json",
```

Comment:
There doesn't seem to be any constants for MIME types; should we pull one out for this extremely common one?